src/debug.cc

```diff
@@ -207,15 +208,17 @@ _db_print_stderr(const char *format, va_list args)
 
 #if HAVE_SYSLOG
 static void
-_db_print_syslog(const char *format, va_list args)
+_db_print_syslog(bool force, const char *format, va_list args)
```

Comment:
s/force/forceAlert/ to be slightly more precise and to ease searching for this parameter as the reader looks through its other uses in debug.cc.

You may also want to mark this parameter as `const`.